generated/java/build.gradle

```diff
@@ -124,14 +122,14 @@ project(':generated:java:google-cloud-logging-v2') {
 
 project(':generated:java:google-cloud-pubsub-v1') {
   dependencies {
-    compile 'com.google.api.grpc:grpc-google-cloud-pubsub-v1:' + generated_pubsub_version
+    compile 'com.google.api.grpc:grpc-google-cloud-pubsub-v1:' + generated_grpc_version
   }
 }
 
 project(':generated:java:google-cloud-language-v1') {
```

Comment:
Speech and language have their versions in separate packages; we can fix in this PR or submit separately.